keadm/cmd/keadm/app/keadm.go

```diff
@@ -18,17 +18,33 @@ package app
 
 import (
 	"flag"
+	"fmt"
 
 	"github.com/spf13/pflag"
+	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/apiserver-network-proxy/pkg/util"
 
 	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd"
 )
 
 // Run executes the keadm command
 func Run() error {
-	flagSet := pflag.NewFlagSet("keadm", pflag.ExitOnError)
-	flagSet.AddGoFlagSet(flag.CommandLine)
-
+	flagSet := flag.NewFlagSet("keadm", flag.ExitOnError)
 	cmd := cmd.NewKubeedgeCommand()
+	flags := cmd.Flags()
+	klog.InitFlags(flagSet)
+	err := flagSet.Set("v", "0")
```

Comment:
Is this default value must required?

Reply:
> Is this default value must required?

yes

Reply:
This will bring out all klog flags, do we need to hide some?

Reply:
> This will bring out all klog flags, do we need to hide some?

There will be no other klog flags introduced here, only "- v, -- v Level number for the log level verbosity"